pyproject.toml

```diff
@@ -93,6 +93,7 @@ types-psutil = "~5.9.5.20240205"
 types-python-dateutil = "~2.9.0.20240316"
 
 [tool.poetry.extras]
+all = ["beautifulsoup4", "lxml", "html5lib", "curl-cffi", "playwright"]
```

Comment:
```suggestion
all = ["beautifulsoup4", "lxml", "html5lib", "curl-impersonate", "playwright"]
```

Reply:
hmm, that's the thing, the package is called `curl-cffi` but our extra is called according to the client -> `curl-impersonate` ... should we align it to `curl-cffi`?

Reply:
in case of `beautifulsoup4` the extra is also called just `beautifulsoup`

Reply:
Nah, this is better. Sorry.